### PR TITLE
Hello Rails with React

### PR DESCRIPTION
**Pull request by PrangonGhose**
### Changes made in this PR
- A sample website is setup where each refresh can show a random greetings message to the user.
- Built with `Rails`, `React`, and `Redux`.
- Data from backend is brought to the store using `fetch API`.
- React components and Redux store is used to made data from API available on global scope.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-ruby "3.1.3"
+ruby '3.1.3'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.4", ">= 7.0.4.3"
-gem "webpacker"
+gem 'rails', '~> 7.0.4', '>= 7.0.4.3'
+gem 'webpacker'
 
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
```
Comment (hiromataba) [inline Gemfile:11]: - [ ] Please use `jsbundling-rails` gem as required for this project

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @PrangonGhose    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PrangonGhose** [comment]
### Changes made after first review
- Added `jsbundling-rails` gem.

**Review by victorokoroji — APPROVED** [review]
Hi @PrangonGhose, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**